docs/bigquery-usage.rst

```diff
@@ -361,31 +359,22 @@ Poll until the job is complete:
 Inserting data (synchronous)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Load data synchronously from a local CSV file into a new table.  First,
-create the job locally:
+Load data synchronously from a local CSV file into a new table:
 
 .. doctest::
 
+   >>> import csv
    >>> from gcloud import bigquery
+   >>> from gcloud.bigquery import SchemaField
    >>> client = bigquery.Client()
    >>> table = dataset.table(name='person_ages')
+   >>> table.schema = [
+   ...     SchemaField(name='full_name', type='string', mode='required'),
+   ...     SchemaField(name='age', type='int', mode='required)]
    >>> with open('/path/to/person_ages.csv', 'rb') as file_obj:
-   ...     job = table.load_from_file(
-   ...         file_obj,
-   ...         source_format='CSV',
```

Comment:
Nope, I figure to update the usage doc as I add the feature (since I'm not sure how it will be spelled yet).